src/components/shared/form_fields/CityAutocompleteField.vue

```diff
@@ -85,6 +86,7 @@ const ariaDescribedby = useAriaDescribedby(
 	`${props.inputId}-error`,
 	computed<boolean>( () => props.showError )
 );
+const scrollIntoView = useAutocompleteScrollIntoViewOnFocus( props.scrollTargetId, 769 );
```

Comment:
This threshold number is also duplicated across fields and makes my "magic number" code smell receptors tingle and from the function call it's also not clear what the number means. Maybe you can export it as a const (e.g. `DEFAULT_SMALL_SCREEN_THRESHOLD`) in the composable, so it's also clear where it's coming from. The bundler will hopefully optimize that away.